src/routes/+page.svelte

```diff
@@ -254,8 +174,8 @@
 		grid-area: menu;
 	}
 
-	.grafiek {
-		grid-area: grafiek;
+	.trash-graph {
```

Comment:
Deze classname is specifieker dan grafiek maar misschien zou hij nog iets specifieker kunnen? Zoals bijv bij trash-removed-last-month?

Reply:
Goed gezien, ik heb het nu trash-collected-over-time-graph genoemd